reference/configuration/framework.rst

```diff
@@ -248,18 +247,25 @@ you use PHPstorm on the Mac OS platform, you will do something like:
             'ide' => 'phpstorm://open?file=%%f&line=%%l',
         ));
 
+.. versionadded:: 3.2
+
+    When running your app in a container or in a virtual machine, you can tell
+    Symfony to map files from the guest to the host by changing their prefix.
+    This map should be specified at the end of the URL template::
+
+        // /guest/.../file will be opened as /host/.../file on the host
+        // and /foo/.../file as /bar/.../file also
+        'myide://%f:%l#/guest/=/host/&/foo/=/bar/&...'
```

Comment:
I think we should explain the format a bit more. Afaics, you have to differentiate the URL template and the prefixes with the `#` character? (should be explained)

Also, `/guest/` seem to indicate that's a regex?